### DISCUSSION TITLE
How add dependency of `setup.py` based project

### DISCUSSION THREAD

**Discussion by chatblanc-ciel** [astral-sh/rye#458] [Q&A]
These projects are not enable to add dependency.

e.g. 

- https://github.com/aboSamoor/pycld2

# When sync

```shell
ERROR: Could not build wheels for pycld2, which is required to install pyproject.toml-based projects
```

**Comment by mitsuhiko** [comment]
It would appear this is an issue with the package you are trying to use: https://github.com/aboSamoor/pycld2/issues/23

**Reply by chatblanc-ciel** [reply]
Thank you for reply. But all of my dependencies are not enabling to use.
